src/DependencyInjection/Configuration.php

```diff
@@ -396,6 +398,88 @@ private function getFlexibleSslNode(): ArrayNodeDefinition
+                                    })
+                                ->end()
+                                ->children()
+                                    ->enumNode('value')
+                                        ->values(['unsafe-none', 'require-corp', 'credentialless'])
+                                        ->info('The COEP policy value')
+                                    ->end()
+                                    ->booleanNode('report_only')
+                                        ->defaultFalse()
+                                        ->info('Use Cross-Origin-Embedder-Policy-Report-Only header instead of Cross-Origin-Embedder-Policy')
+                                    ->end()
+                                    ->scalarNode('report_to')
+                                        ->defaultNull()
+                                        ->info('Reporting endpoint name for COEP violations (requires Reporting API configuration)')
+                                    ->end()
+                                ->end()
+                            ->end()
+                            ->arrayNode('coop')
+                                ->info('Cross-Origin-Opener-Policy (COOP) header. Can be a string (unsafe-none, same-origin-allow-popups, same-origin, noopener-allow-popups) or an array with value, report_only, and report_to')
+                                ->beforeNormalization()
+                                    ->ifString()
+                                    ->then(static function (string $v): array {
+                                        return ['value' => $v];
+                                    })
+                                ->end()
+                                ->children()
+                                    ->enumNode('value')
+                                        ->values(['unsafe-none', 'same-origin-allow-popups', 'same-origin', 'noopener-allow-popups'])
+                                        ->info('The COOP policy value')
+                                    ->end()
+                                    ->booleanNode('report_only')
+                                        ->defaultFalse()
+                                        ->info('Use Cross-Origin-Opener-Policy-Report-Only header instead of Cross-Origin-Opener-Policy')
+                                    ->end()
+                                    ->scalarNode('report_to')
+                                        ->defaultNull()
+                                        ->info('Reporting endpoint name for COOP violations (requires Reporting API configuration)')
+                                    ->end()
+                                ->end()
+                            ->end()
+                            ->arrayNode('corp')
+                                ->info('Cross-Origin-Resource-Policy (CORP) header. Can be a string (same-site, same-origin, cross-origin) or an array with value')
```

Comment:
Do we actually need to use an array with a single `value` key here ? I would use the enumNode on `corp` directly (we can still change this in the future if we need to add more settings).

Reply:
You are right. I moved it back to an enumNode